spark/src/main/scala/geotrellis/spark/io/RasterReader.scala

```diff
@@ -64,13 +64,13 @@ object RasterReader {
     def readFully(byteReader: ByteReader, options: Options) = {
       val geotiff = SinglebandGeoTiff(byteReader)
       val raster: Raster[Tile] = geotiff.raster
-      (ProjectedExtent(raster.extent, options.crs.getOrElse(geotiff.crs)), raster.tile)
+      (ProjectedExtent(raster.extent, options.crs.getOrElse(geotiff.crs)), raster.tile.toArrayTile)
```

Comment:
@pomadchin I'm curious; before these changes, were we benefitting from any lazy evaluation? If so, another option to making the classes Java serialization friendly would be to override the `writeObject` or `writeResolve` (I forget the difference) to implicitly do the `toArrayTile` part when needed.

Reply:
it would be enough to extend serializable, not sure what would be the benefit of implementing these method(s).